app/views/directives/centreGender.html

```diff
@@ -39,10 +39,17 @@
       <td ng-bind="prebookingreceived"></td>
       <td ng-bind="contingency"></td>
     </tr>
-    <tr ng-if="expectedin !== undefined">
-      <td>Expected incoming</td>
+
+    <tr ng-if="expectedin !== undefined" ng-click="showIncoming = expectedin.length>0 && !showIncoming"
+        ng-class="expectedin.length>0 ? showIncoming ? 'close' : 'open' : ''">
+      <td>
```

Comment:
that's not a safe assumption
it being defined and being empty are two different states